posts/jhipster-5-release.md

```diff
@@ -0,0 +1,218 @@
+---
+authors:
+- Kenneth Hegeland, Theo Lebrun, Raphael Brugier
+tags:
+- Brace yourself, JHipster 5 is out!
+date: 2018-06-23T15:43:55.000Z
+title: "JHipster 5"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2018/06/JHipster-5-banner.png
+---
+
+JHipster 5 has just been released, and we thought it would be nice to write a post compiling all the new features. Throughout this post, we will be linking to the official docs, which provide some more detailed explanations of how to use these new features.
+
+With this new release, we think that beginners and experts alike will be excited. We have new technologies being introduced, upgraded versions, and quality of life improvements. Let’s begin with front-end changes.
+
+
+# Frontend changes
+
+## React support
+
+Frontend technologies move fast, and it’s at times difficult to keep up with what is currently the most popular framework. 
+Largely, in the past, Angular has dominated as one of the most popular front-end frameworks, but recently, popularity has been shifting towards React. 
+As of this writing, React has 97,250 stars on GitHub, and Angular has 36,800 stars - while this isn’t proof, it does show that popularity is changing a bit, so it may become important as a developer to learn both technologies. 
+Personally, I find that JHipster is an amazing learning tool, as well as speeding professional development speed, so whether you are new to React, or a veteran, you will find this feature useful. 
+
+If you fall into the former camp (new to React), well then familiarity with JHipster will greatly increase the time it takes to learn React. 
+JHipster for Angular already taught us how to structure the application, how API calls are made, authentication is handled, etc. 
+So now with a JHipster React app, we have a perfect guideline for learning. We can now quickly catch up on React, based on our existing knowledge of JHipster and Angular.
+
+If we are a veteran React developer, then you will benefit greatly in two possible ways. 
+First it is faster to setup a new project with a Java backend. As always with JHipster, if you know the technology stack, then you are able to very quickly set up a project and begin delivering business value. 
+It is not surprising this also occurs when generating a React application. 
+Second is another great JHipster benefit, you can quickly learn things that JHipster provides, that may be new to you.
+If you are new to JHipster, then you will get to see an _opinionated_ approach to React.
+
+If you are entirely unfamiliar with JHipster, we still have good news!
```

Comment:
I would move this paragraph out of the React section and may include it as part of the intro

Reply:
good point. this is more general, moving this to intro.